tools/sync_test_server/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6.11.4
+FROM node:10
```

Comment:
why the update to 10?

Reply:
We dropped support for node 6 some time ago.